src/main/java/controller/UserController.java

```diff
@@ -0,0 +1,74 @@
+package controller;
+
+import db.Database;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import util.RequestParser;
+
+import java.util.Objects;
+
+public class UserController {
+
+        private Logger log = LoggerFactory.getLogger(getClass());
+        // 인스턴스 변수
+        private static UserController instance;
+
+        // private 생성자
+        public UserController() {
+
+        }
+
+        // 인스턴스 반환 메소드
+        public static UserController getInstance() {
+            if (instance == null) {
+                synchronized (UserController.class) {
+                    if (instance == null) {
+                        instance = new UserController();
+                    }
+                }
+            }
+            return instance;
+        }
+
+        public String mapToFunctions(String[] parsedUrl) {
```

Comment:
```suggestion
        public String mapToFunctions(String[] parsedUrl) {
```
용도가 다른 매개변수를 배열로 사용하는 일은 별로 없습니다. 각각을 별도의 매개변수로 분리하는 게 좋을 것 같아요.